ignite/services/chain/generate.go

```diff
@@ -119,7 +121,7 @@ func (c *Chain) Generate(
 		return err
 	}
 
-	c.ev.Send("🛠  Building proto...")
+	c.ev.Send("Building proto...", events.ProgressStarted())
```

Comment:
No more icon for this message ? Or maybe there's a default icon ?

Reply:
In that case the `Send` is updating the spinner text because is using a "status update option" which is the `ProgressStarted` and because of that the end result to the user will be that the text right next to the spinner would look weird, in case of using the icon, because you would have "SPINNER ICON Building proto...". I guess the blue spinner icon is already The icon.

Naturally it is a subjective opinion 😄. Personally I think we are overusing icons, I believe icons are good to help the user and make the UI nicer but shouldn't be overused. There are other resources like a reduced set of colors for different UI elements like links or mnemonics for example, or bold text, etc, that might be more "aligned" with terminal UIs.

In any case I can put them back 👍